wazuh/wazuh-dashboard-plugins
# Refactor top N tables in vulnerabilities dashboard

### Description
This PR improves the titles and columns width tables in the `Vulnerability Detection` Dashboard to avoid truncating titles on small screens.
 
### Issues Resolved
https://github.com/wazuh/wazuh-dashboard-plugins/issues/6353

### Evidence

#### Windows width: 1920px
![image](https://github.com/wazuh/wazuh-dashboard-plugins/assets/103193307/18145472-be30-4ded-a401-055ec182465e)

#### Windows width: 1440px
![image](https://github.com/wazuh/wazuh-dashboard-plugins/assets/103193307/1cba022d-b233-43b8-a829-abec7d1b8b7f)

#### Windows width: 1024px
![image](https://github.com/wazuh/wazuh-dashboard-plugins/assets/103193307/74bca61e-1ec3-4533-ab3c-ac01ff4df872)

#### Windows width: 1000px
![image](https://github.com/wazuh/wazuh-dashboard-plugins/assets/103193307/94bb6ccb-2bc8-48e1-a1a1-712cc67b9aae)

#### Windows width: 767px
![image](https://github.com/wazuh/wazuh-dashboard-plugins/assets/103193307/8f81e2d4-9123-4057-948a-20cbd7d65e93)

### Test

1. Run script: `scripts/vulnerabilities-events-injector/dataInjectScript.py` to generate vulnerabilities.
2. Go to `App Settings -> Miscellaneous` and run the health check.
3. Go to `Vulnerability Detection` and check the top tables in the Dashboard.

### Check List
